client/src/components/Icons/SVGIcon.js

```diff
@@ -0,0 +1,103 @@
+        />
+      );
+    case "email":
+      return (
+        <path
+          {...props}
+          d="M17.4166 4.74996C17.4166 3.87913 16.7041 3.16663 15.8333 3.16663H3.16665C2.29581 3.16663 1.58331 3.87913 1.58331 4.74996V14.25C1.58331 15.1208 2.29581 15.8333 3.16665 15.8333H15.8333C16.7041 15.8333 17.4166 15.1208 17.4166 14.25V4.74996ZM15.8333 4.74996L9.49998 8.70829L3.16665 4.74996H15.8333ZM9.49998 10.2916L3.16665 6.33329V14.25H15.8333V6.33329L9.49998 10.2916Z"
+        />
+      );
+    case "instagram":
+      return (
+        <path
+          {...props}
+          d="M11.4331 0H2.56661C1.15139 0 0 1.15139 0 2.56661V11.4331C0 12.8483 1.15139 14 2.56661 14H11.4331C12.8483 14 14 12.8486 14 11.4331V2.56661C14 1.15139 12.8486 0 11.4331 0ZM6.99984 11.6087C4.45875 11.6087 2.39128 9.54125 2.39128 6.99983C2.39128 4.45875 4.45875 2.39128 6.99984 2.39128C9.54125 2.39128 11.6087 4.45875 11.6087 6.99983C11.6087 9.54092 9.54092 11.6087 6.99984 11.6087ZM11.7569 3.34414C11.1553 3.34414 10.6661 2.85495 10.6661 2.25373C10.6661 1.65252 11.1553 1.16332 11.7569 1.16332C12.3581 1.16332 12.8473 1.65252 12.8473 2.25373C12.8473 2.85495 12.3581 3.34414 11.7569 3.34414Z"
+        />
+      );
+    case "calendar":
+      return (
+        <path
+          {...props}
+          d="M14.25 3.16671H15.0417C15.9125 3.16671 16.625 3.87921 16.625 4.75004V15.8334C16.625 16.7042 15.9125 17.4167 15.0417 17.4167H3.95833C3.07958 17.4167 2.375 16.7042 2.375 15.8334V4.75004C2.375 3.87921 3.07958 3.16671 3.95833 3.16671H4.75V1.58337H6.33333V3.16671H12.6667V1.58337H14.25V3.16671ZM3.95833 7.91671V15.8334H15.0417V7.91671H3.95833ZM15.0417 6.33337H3.95833V4.75004H15.0417V6.33337ZM9.5 9.50004H5.54167V13.4584H9.5V9.50004Z"
+        />
+      );
+    case "time":
+      return (
+        <path
+          {...props}
+          d="M8.9925 1.5C4.8525 1.5 1.5 4.86 1.5 9C1.5 13.14 4.8525 16.5 8.9925 16.5C13.14 16.5 16.5 13.14 16.5 9C16.5 4.86 13.14 1.5 8.9925 1.5ZM9 15C5.685 15 3 12.315 3 9C3 5.685 5.685 3 9 3C12.315 3 15 5.685 15 9C15 12.315 12.315 15 9 15ZM8.25 5.25H9.375V9.1875L12.75 11.19L12.1875 12.1125L8.25 9.75V5.25Z"
+        />
+      );
+    case "facebook":
+      return (
+        <path
+          {...props}
+          d="M11.9134 9.23724H10.2146V15.0482H7.64094V9.23724H6.41687V7.19503H7.64094V5.87349C7.64094 4.92845 8.1217 3.44861 10.2376 3.44861L12.144 3.45606V5.43837H10.7608C10.5339 5.43837 10.2148 5.54422 10.2148 5.99503V7.19693H12.1382L11.9134 9.23724Z"
+        />
+      );
+    default:
+      return <path />;
+  }
+};
+
+const SVGIcon = ({
+  name = "",
+  style = {},
+  fill = "#764437",
+  viewBox = "",
+  width = "19",
+  className = "",
+  height = "19",
+  fillOpacity="0.95"
+}) => (
+  <svg
+    width={width}
+    style={style}
+    height={height}
+    className={className}
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox={viewBox || getViewBox(name)}
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+  >
+    {getPath(name, { fill })}
+  </svg>
+);
```

Comment:
See Roger's comments re Font Awesome. Ideally someone could just pass in a prop for the icon they want (like `facebook`) and then this would render the Font Awesome Facebook icon